fix(apple): Add SentrySDK dependency with xCode (#4674)
getsentry/sentry-docs: src/docs/clients/cocoa/index.mdx

Starting with sentry-cocoa version `4.4.3`, we support Swift Package Manager.

> To integrate Sentry into your Xcode project using SPM, open your App in Xcode and open File -> Add Packages. Then add sentry-cocoa by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` on the top-right search field. Define you dependecy rule by selecting the SDK version (or branch), then press "Add Package" button.

Reviewer: ```suggestion
To integrate Sentry into your Xcode project using SPM, open your App in Xcode and open File -> Add Packages. Then add sentry-cocoa by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git` in the top right search field. Define your dependency rule by selecting the SDK version (or branch), and then click the "Add Package" button.
```
